tools/deployment/osqueryd.service

```diff
@@ -13,6 +13,7 @@ ExecStart=/usr/bin/osqueryd \
 Restart=on-failure
 KillMode=control-group
 KillSignal=SIGTERM
+CPUQuota=10%
```

Comment:
How does this work with: `WatchdogLimitType::UTILIZATION_LIMIT` setting?

Reply:
For systems with a systemd >= version 213 and with the proper kernel configuration `CONFIG_CFS_BANDWIDTH` then CPUQuota is the first line of defense. If somehow the quota is violated then nothing really happens, but if the process is really "out of control" then the osquery watchdog is a second line of defense.

For all other systems, the osquery watchdog is the first line of defense.

In addition to this proposed change, and the watchdog's enforcement, osquery also nices itself to the lowest priority and applies any background-IO preferences available.

Reply:
We could also move this PR to set 20% as the quota. That would make osquery the first line of defense then systemd/cgroups the second.